connect/governance/governance.py

```diff
@@ -318,6 +326,12 @@ def register(
 
             self.clear_evidence()
 
+    def _parse_assessment_plan_url(self, assessment_plan_url):
+        api_base = assessment_plan_url.split("/use_cases/")[0]
+        result = urlparse(assessment_plan_url)
+        query = parse_qs(result.query)
+        return api_base, query.get("access_token", [None])[0]
```

Comment:
Access token in a GET parameter is a security issue unless this is a one time token and/or has a short lifetime.